Gemfile

```diff
@@ -14,6 +14,11 @@ gem 'coffee-rails', '~> 4.2'
 gem 'jquery-rails'
 gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
+gem 'travis'

```

Comment:
Travis could also go in the dev/test group (I believe).

Not needed to load in production 😀